examples/segmentation_3d/unet_training_dict.py

```diff
@@ -121,16 +122,16 @@ def main():
 
     # start a typical PyTorch training
     val_interval = 2
-    best_metric = -1
+    best_metric: float = -1.0
     best_metric_epoch = -1
-    epoch_loss_values = list()
-    metric_values = list()
+    epoch_loss_values: List[float] = list()
```

Comment:
I am not very familiar with type hints, when to use `list` and when to use `List`?

Reply:
`list` is one specific object type and must match that exactly.  "List" is a generic "List-like" type definition so that non-`list` objects that quack like a list are acceptable.